### ISSUE TITLE
creating zones does not work

### ISSUE THREAD

**Issue by exaSR** [dmustanger/7dtd-ServerTools#121]
# problem(s):
* "zone new" responds with "circle or save" as expected
* however, neither "zone circle" nor "zone save" send any replies. Instead an error is logged:
```
[SERVERTOOLS] Error in ZoneSetupConsole.Run: System.IndexOutOfRangeException: Array index is out of range.
 at ServerTools.ZoneSetupConsole.Execute (System.Collections.Generic.List`1 _params, CommandSenderInfo _senderInfo) [0x00000] in <filename unknown>￼ .
```
* "zone back" is not accepted at all

# analysis
Sadly, I'm not a C#-guy, so I can't create a proper pull request; but I did some code review and found the following in ZoneSetupConsole.cs:
1. bracket error puts the "back" test inside the "save" conditional...
2. "new" seems to create arrays of length zero (_blank and _blank2)
3. "circle" and "save" try to write into that array, probably causing the error above.

**Comment by ObsComp** [comment]
I take it back. After some tests I see no response from zones. I will put it into the test lab and figure it out.

**Comment by ObsComp** [comment]
I have it fixed in the next release 11.3
